src/Components/CreateImageWizard/validators.ts

```diff
@@ -33,13 +33,23 @@ export const isGcpEmailValid = (gcpShareWithAccount: string | undefined) => {
 };
 
 export const isMountpointMinSizeValid = (minSize: string) => {
+  if (!minSize) {
+    return false;
+  }
```

Comment:
This might be fine here as 'Must be larger than 0' is kinda fitting for both invalid and missing value, but ... (continuing in a comment lower)